src/doc/unstable-book/src/library-features/global-asm.md

```diff
@@ -24,15 +24,15 @@ conventions of the assembler in your toolchain.
 
 A simple usage looks like this:
 
-```rust,ignore
+```rust,ignore (requires-external-file)
 # #![feature(global_asm)]
-# you also need relevant target_arch cfgs
+# // you also need relevant target_arch cfgs
 global_asm!(include_str!("something_neato.s"));
 ```
 
 And a more complicated usage looks like this:
 
-```rust,ignore
+```rust,no_run
```

Comment:
This failed in https://github.com/rust-lang/rust/pull/81748#issuecomment-773448498: https://github.com/rust-lang-ci/rust/runs/1831587205